cvmfs/cvmfs.cc

```diff
@@ -1331,6 +1331,25 @@ static void cvmfs_getxattr(fuse_req_t req, fuse_ino_t ino, const char *name,
     attribute_value = string(VERSION) + "." + string(CVMFS_PATCH_LEVEL);
   } else if (attr == "user.pubkeys") {
     attribute_value = mount_point_->signature_mgr()->GetActivePubkeys();
+  } else if (attr == "user.repo_counters") {
+    attribute_value = mount_point_->catalog_mgr()->GetRootCatalog()->
+                                    GetCounters().GetCsvMap();
+  } else if (attr == "user.repo_metainfo") {
```

Comment:
I think it makes sense to move the code of this `if` branch into a helper function